roles/neutron-data-network/tasks/main.yml

```diff
@@ -262,6 +278,24 @@
         - /etc/neutron/services/loadbalancer/haproxy/lbaas_agent.ini
         - /etc/neutron/neutron_lbaas.conf
 
+- name: install neutron lbaas service (rhel distro)
+  systemd_service:
```

Comment:
We can probably change the dest path on where we drop these templates to where the OSP packages expect. This is what we had to do for horizon as well. IMO I think it's be better if we could just use the OSP packages as is w/out having to customize the systemd stuff.